.github/workflows/ci-arm64.yml

```diff
@@ -0,0 +1,79 @@
+name: Build OpenSSL Arm64
+
+on:
+  pull_request:
+    branches: [ "master" ]
+  workflow_dispatch:
+    inputs:
+      openssl_build:
+        description: 'OpenSSL Arm64'
+        required: false
+        default: true
+        type: boolean
+      openssl_clang_build:
+        description: 'OpenSSL with clang-cl Arm64'
+        required: false
+        default: true
+        type: boolean
+
+jobs:
+  build-openssl-arm64:
+    runs-on: [self-hosted, Windows, ARM64, GCC]
+    timeout-minutes: 600
+
+    steps:
+    
+    - name: Build OpenSSL Arm64
+      shell: cmd
+      run: |
+        echo on
+        rmdir /s /q openssl
+        git clone https://github.com/openssl/openssl.git --single-branch --depth 1
+        cd openssl
+        call "%ProgramFiles%\Microsoft Visual Studio\2022\Community\VC\Auxiliary\Build\vcvarsall.bat" arm64
+        set VSCMD_DEBUG=3
+        set PATH=%PATH%;%VCINSTALLDIR%\Tools\Llvm\ARM64\bin
+        perl Configure VC-WIN64-ARM
+        nmake
+        nmake test
+        .\apps\openssl.exe speed 2> benchmark.txt
+        type benchmark.txt
+
+  build-openssl-clang-arm64:
+    runs-on: [self-hosted, Windows, ARM64, GCC]
+    timeout-minutes: 600
+
+    steps:
+
+    - name: Build OpenSSL with clang-cl Arm64
+      shell: cmd
+      run: |
+        echo on
+        rmdir /s /q openssl
+        git clone https://github.com/openssl/openssl.git --single-branch --depth 1
+        cd openssl
+        call "%ProgramFiles%\Microsoft Visual Studio\2022\Community\VC\Auxiliary\Build\vcvarsall.bat" arm64
+        set VSCMD_DEBUG=3
+        set PATH=%PATH%;%VCINSTALLDIR%\Tools\Llvm\ARM64\bin
+        perl Configure VC-CLANG-WIN64-CLANGASM-ARM
+        nmake
+        nmake test
+        .\apps\openssl.exe speed 2> benchmark.txt
+        type benchmark.txt
+
+  build-openssl-aarch64:
```

Comment:
It's not straightforward from the job and steps name that this is a Linux build vs. the previous Windows builds. I'd suggest to state "Windows" and "Linux" in that jobs explicitly.

Reply:
Changed